opa-fig-autocomplete/go.mod

```diff
@@ -0,0 +1,62 @@
+module github.com/opa/contrib
```

Comment:
[nit] Can we make this the Github URL? just to be less confusing, like
```suggestion
module https://github.com/open-policy-agent/contrib/opa-fig-autocomplete
```

Reply:
thank you 🙋🏻‍♂️